rnd/autogpt_server/autogpt_server/blocks/talking_head.py

```diff
@@ -0,0 +1,145 @@
+import time
+from typing import Literal
+
+import requests
+
+from autogpt_server.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from autogpt_server.data.model import BlockSecret, SchemaField, SecretField
+
+
+class ClipsBlock(Block):
```

Comment:
pls more descriptive name :)

Reply:
I'd second this, the builder is currently struggling with un clear names. It is currently the only thing the are searched on so needs to be super clear.